dev/io.openliberty.jaxws.config_fat/publish/files/GzipInterceptorsTestServer/server.xml

```diff
@@ -0,0 +1,15 @@
+<server>
+    <featureManager>
+        <feature>jsp-2.2</feature>
+        <feature>jaxws-2.2</feature>
```

Comment:
Looking at the build failures, I wonder if this feature list is causing the break. Should verify that the JSP-2.2 feature is the right one to pull in for this test. 

```
Runtime features were not of the expected version for repeat action (Server: EnableDefaultValidationTestServer, Action: EE8_FEATURES).
Expected: servlet-4.0, Actual: servlet-3.1.
```

Reply:
Changes are unrelated, you can ignore this.